ably-ios/ARTPayload.m

```diff
@@ -317,6 +326,9 @@ - (ARTStatus *)decode:(ARTPayload *)payload output:(ARTPayload *__autoreleasing
 
 @end
 
+
+// MARK: Cipher
```

Comment:
It helps with the `Show Document Items`:
<img width="548" alt="screen_shot_2016-01-14_at_12_35_34" src="https://cloud.githubusercontent.com/assets/3541185/12324588/973527fe-babb-11e5-878d-c65b0ab333b9.png">